.buildconfig.yml

```diff
@@ -23,6 +23,10 @@ projects:
     path: components/concept/awesomebar
     description: 'An abstract definition of an awesomebar component.'
     publish: true
+  lib-auth:
```

Comment:
nit: If you're keeping lib, can we move this down to the other libs modules, alphabetically?

Reply:
Done